Lib/bdb.py

```diff
@@ -194,6 +277,12 @@ def user_exception(self, frame, exc_info):
         pass
 
     def _set_stopinfo(self, stopframe, returnframe, stoplineno=0):
+        """Set the attributes for stopping.
+
+        If `stoplineno` is greater than or equal to 0, then stop at line
```

Comment:
I think we can also remove the backticks in `checkfuncname()`, so we'll have  all the docstrings formatted consistently.